src/services/github/commit.ts

```diff
@@ -0,0 +1,136 @@
+import { GitHubAPI } from "probot";
+import { getLogger } from "../../config/logger";
+import { getCommits as getCommitsQuery } from "../../sync/queries";
+
+const logger = getLogger("services.github.commits");
+
+export const getSpecificGithubCommits = async (github: GitHubAPI, params: SpecificCommitParams): Promise<GithubCommit[]> => {
+	if (!params.commitRefs?.length) {
+		return [];
+	}
+
+	try {
+		const response = (await github.graphql(
```

Comment:
nitpick: maybe put the GraphQL query into a variable like the other ones.

Reply:
In this case, we can't as we're doing string interpolation.